app_dart/test/request_handlers/refresh_github_commits_test.dart

```diff
@@ -131,6 +138,7 @@ void main() {
       expect(db.values.values.whereType<Commit>().length, 2);
       expect(db.values.values.whereType<Task>().length, 10);
       expect(db.values.values.whereType<Commit>().map<String>(toSha), <String>['1', '3']);
+      expect(db.values.values.whereType<Commit>().map<int>(toTimestamp), <int>[nowMillionSecond, nowMillionSecond]);
```

Comment:
Why isn't this two difrerent timestamps?

Reply:
Was using one same timestamp for all commits. Now changed to different timestamp for different commits.